django/api/templates/admin/api/initiatedgoelectricrebateapplication/change_form.html

```diff
@@ -0,0 +1,18 @@
+{% extends "admin/change_form.html" %} {% load i18n admin_urls jazzmin %} {%
+get_jazzmin_ui_tweaks as jazzmin_ui %} 
+
+ {% block field_sets %}
+                    <div class="col-12 col-lg-9">
+                        <div class="card card-primary card-outline">
+                            <div class="card-header">
+                                <div class="h3">
+                                   <span class="text-bold">{{original.application_type}}</span> application {{original.id}}
```

Comment:
all household_initiated applications are of type household; not sure if we even need to override the change_form template in this case; depending on what the "title" is... see original here: https://github.com/farridav/django-jazzmin/blob/v2.5.0/jazzmin/templates/admin/change_form.html

Reply:
I think you can just remove it.

Reply:
Does it break anything if you remove it?